### PR TITLE
Add "Delivered on" column to TSP queue.

### PR DESCRIPTION
**Pull request by garrettqmartin8**
## Description

As a TSP user
I want to see the delivered on date
So that I can identify which shipments have been recently delivered and may need to be invoiced.

## Setup

- Log in as a TSP user
- Take a look at the "Delivered on" column that is now in the queue table

## Code Review Verification Steps

* [x] User facing changes have been reviewed by design.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/164788590) for this change

## Screenshots
Screen Recording 2019-04-01 at 11 29 47

### CODE REVIEW

File: src/scenes/TransportationServiceProvider/QueueTable.jsx [hunk 1]
```diff
@@ -118,6 +118,22 @@ class QueueTable extends Component {
                   d.original_pickup_date,
```
Comment (donaldthai) [inline src/scenes/TransportationServiceProvider/QueueTable.jsx]: Was there a merge issue because I see this is the same as line 128?

Comment (donaldthai) [reply src/scenes/TransportationServiceProvider/QueueTable.jsx]: There are two pickup date columns?

Comment (garrettqmartin8) [reply src/scenes/TransportationServiceProvider/QueueTable.jsx]: Good catch!

**Comment by garrettqmartin8** [comment]
Addressed @donaldthai's comment.

**Pull request merged**